src/pages/components/SecondaryBar.qml

```diff
@@ -41,6 +41,21 @@ Column {
         }
     }
 
+    OverlayListItem {
+        height: overlay.toolBar.toolsHeight
+
+        iconWidth: root.iconWidth
+        horizontalOffset: root.horizontalOffset
+        //% "History"
+        text: qsTrId("sailfish_browser-la-history")
+        iconSource: "image://theme/icon-m-backup"
```

Comment:
Lemme check this one as well. Would be better to have the icon duplicated so that we'd have something that would depict history/time i.e. would not refer to the backup.